app/views/assessments/index.html.erb

```diff
@@ -65,7 +65,7 @@
             <%= link_to "<i class='material-icons left'>grading</i> Gradebook".html_safe, [@course, @cud, :gradebook], title: "View your gradebook", :class => "btn btn-large red darken-3" %>
           <% end %>
 
-          <% if @cud.instructor? %>
+          <% if (@cud.instructor? || @cud.course_assistant? && @course.watchlist_configuration.allow_ca) %>
```

Comment:
It can't be assumed that every course has an existing `watchlist_configuration`, especially if they have never updated the settings before.

![Screenshot 2022-11-25 at 02 10 34](https://user-images.githubusercontent.com/9074856/203922148-5f042a04-1915-4cd3-8c25-f58de020e175.png)

Consider using Ruby's safe navigation operator (`@course.watchlist_configuration&.allow_ca`)

Reply:
Makes sense with regards to the case that every course might not have an existing watchlist_configuration.

But it seems like `@course.watchlist_configuration&.allow_ca` would return nil, and there are many behaviors with regards to (nil && / || boolean variables). So I will address this in a different way